### PR TITLE
Fix for upgraded thriftrw

### PR DESCRIPTION
**Pull request by jcorbin**
Travis being down yesterday masked this failure.

r @kriskowal @Raynos

### CODE REVIEW

File: hyperbahn/endpoints/index.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ function setupEndpoints(clients, services) {
     /*eslint no-unused-vars: 0*/
     // thrift health endpoint
     var tchannelAsThrift = new TChannelAsThrift({
-        source: '',
+        source: '// lul',
```
Comment (kriskowal) [inline hyperbahn/endpoints/index.js]: https://github.com/uber/thriftrw/issues/51

**Comment by kriskowal** [comment]
there is a stamp. the stamp is green.

File: node/tcollector/tcollector.thrift [hunk 2]
```diff
@@ -2,9 +2,9 @@
 // `Endpoint`, will be the server being hit by the call.
 
 struct Endpoint {
-    1: i32 ipv4
-    2: i32 port
-    3: string serviceName
```
Comment (Raynos) [inline node/tcollector/tcollector.thrift]: You cannot do this. Revert

Comment (Raynos) [inline node/tcollector/tcollector.thrift]: This is a breaking change.

Comment (Raynos) [inline node/tcollector/tcollector.thrift]: Just set strict: false

**Comment by Raynos** [comment]
Verified this change in python & node. Its safe. Going to change the flashlight/tcollector backend.

**Pull request merged**